rust/validation/src/validation/store.rs

```diff
@@ -115,6 +123,23 @@ impl StoreValidate<&str> for Store {
             Ok(ctx.into())
         }
     }
+    fn validate_value(
+        &self,
+        value: &mut Value,
+        schema_name: &str,
+        configuration: Option<&Configuration>,
+    ) -> ValidationResult {
+        if let Ok(schema_type) = Schema::try_from(schema_name) {
+            self.validate_value(value, schema_type, configuration)
```

Comment:
so here we now have a schema_type that is the schema_name for the other implementation :) I understand whats going on and thats just variable names but :p